Documentation/CRDs/Object-Storage/ceph-object-store-crd.md

```diff
@@ -157,7 +157,7 @@ created anew.
 
 ## Health settings
 
-Rook-Ceph will be default monitor the state of the object store endpoints.
+Rook Ceph will be default monitor the state of the object store endpoints.
```

Comment:
Instead of converting "Rook-Ceph" to "Rook Ceph", what about just saying "Rook"? Everything in these docs is only for the ceph operator since we split other providers out to other repos, so I don't think we have nay reason for that clarification. The definition of "Rook" in these docs is the operator that configures Ceph. It's just simpler and more readable that way IMO.
```suggestion
Rook will by default monitor the state of the object store endpoints.
```

Reply:
@travisn So doing this for all `Rook Ceph` mentions or just the ones I changed in the PR?

Reply:
It's fine to just change the ones in this PR where you are replacing "Rook-Ceph", thanks

Reply:
@galexrt planning on this update?

Reply:
@travisn PR has been updated.